### ISSUE TITLE
Error parsing SQLite SELECT statement

### ISSUE THREAD

**Issue by jalcine** [san650/pretty_print_formatter#9]
I keep seeing:
```
ERROR PARSING {1, :sql_lexer, {:illegal, '?'}} -- SELECT s0."value" FROM "settings" AS s0 WHERE (s0."name" = ?1) ORDER BY s0."name" LIMIT 1 ["hCard_note"]
```

in my log output. I'm using SQLite with Ecto.

**Comment by san650** [comment]
@jalcine thanks for reporting the issue. The fix is released in version v0.1.5

**Comment by jalcine** [comment]
> @jalcine thanks for reporting the issue. The fix is released in version v0.1.5

Thanks @san650; you rock!
